public/style.css

```diff
@@ -163,6 +168,8 @@
   color: #fefefe;
   text-shadow: 1px 1px 2px #0a0a0a;
 }
+<<<<<<< HEAD
+=======
```

Comment:
Please make sure to resolve all merge conflicts prior to submission, and that all merge markers are removed.

Reply:
removed merge markers from css files